### PR TITLE
Add expiration time in expectations page

### PR DESCRIPTION
**Pull request by johanah29**

### CODE REVIEW

File: docs/usage/expectations.md [hunk 1]
```diff
@@ -49,3 +49,18 @@ If you have configured manual expectations in your scenario, you will have the o
 
 <!-- screenshot of the screen populated with manual validation to perform -->
 
+### Customize expectations expiration time
+
+Expectations results must be validated within a time limit. Depending on the expectation's type, there is a default expiration time set in the system. Users have two ways to modify that expiration time:
+
+- In the application.properties file
```
Comment (RomuDeuxfois) [inline docs/usage/expectations.md:56]: We should mention the Docker file instead of the application.properties file
User use a docker image to deploy

**Review by RomuDeuxfois — CHANGES_REQUESTED** [review]

File: docs/usage/expectations.md [hunk 2]
```diff
@@ -49,3 +49,18 @@ If you have configured manual expectations in your scenario, you will have the o
...
+
+![Customize expiration time in app.properties](assets/app_properties_customize_expiration_time.png)
+
+A default expiration time is set for technical and human expectations. Users can override them for each type of expectations.
+
+- In the UI
+
+![Customize expiration time in UI](assets/expectation_customize_expiration_time.png)
+
+When creating an expectation, users can set expiration time. The system's default times are set on the form and users can change it or not.
```
Comment (EllynBsc) [inline docs/usage/expectations.md:66]: When creating an expectation, users can set an expiration time. The system's default times are set on the form and users decide to override it.

Comment (EllynBsc) [inline docs/usage/expectations.md:66]:
```suggestion
When creating an expectation, users can set expiration time. The system's default times are set on the form and users can change it or not.
```

File: docs/usage/expectations.md [hunk 3]
```diff
@@ -49,3 +49,16 @@ If you have configured manual expectations in your scenario, you will have the o
 
 <!-- screenshot of the screen populated with manual validation to perform -->
 
+### Customize expectations expiration time
+
+Expectations results must be validated within a time limit. Depending on the expectation's type, there is a default expiration time set in the system. Users have two ways to modify that expiration time:
+
+- In the Docker .env file
+
```
Comment (RomuDeuxfois) [inline docs/usage/expectations.md:57]:
```suggestion
- In the Docker .env file thanks to this variables
| Parameter                                       | Environment variable                            | Default value      | Description                                                            |
|:------------------------------------------------|:------------------------------------------------|:-------------------|:-----------------------------------------------------------------------|
| openbas.expectation.technical.expiration-time   | OPENBAS_EXPECTATION_TECHNICAL_EXPIRATION-TIME   | 21600              | Expiration time for Technical expectation (detection & prevention)      |
| openbas.expectation.detection.expiration-time   | OPENBAS_EXPECTATION_DETECTION_EXPIRATION-TIME   | 21600              | Expiration time for detection expectation                               |
| openbas.expectation.prevention.expiration-time  | OPENBAS_EXPECTATION_PREVENTION_EXPIRATION-TIME  | 21600              | Expiration time for prevention expectation                              |
| openbas.expectation.human.expiration-time       | OPENBAS_EXPECTATION_HUMAN_EXPIRATION-TIME       | 86400              | Expiration time for human expectation (manual, challenge & article)     |
| openbas.expectation.challenge.expiration-time   | OPENBAS_EXPECTATION_CHALLENGE_EXPIRATION-TIME   | 86400              | Expiration time for challenge expectation                               |
| openbas.expectation.article.expiration-time     | OPENBAS_EXPECTATION_ARTICLE_EXPIRATION-TIME     | 86400              | Expiration time for article expectation                                 |
| openbas.expectation.manual.expiration-time      | OPENBAS_EXPECTATION_MANUAL_EXPIRATION-TIME      | 86400              | Expiration time for manual expectation                                  |

```

**Review by RomuDeuxfois — APPROVED** [review]

**Pull request merged**